rmgpy/statmech/rotationTest.py

```diff
@@ -250,129 +252,131 @@ def test_repr(self):
         self.assertEqual(self.mode.inertia.units, mode.inertia.units)
         self.assertEqual(self.mode.symmetry, mode.symmetry)
         self.assertEqual(self.mode.quantum, mode.quantum)
-        
+
     def test_pickle(self):
         """
         Test that a LinearRotor object can be pickled and unpickled with no
         loss of information.
         """
-        import cPickle
-        mode = cPickle.loads(cPickle.dumps(self.mode,-1))
+        import pickle
```

Comment:
Should this import be relocated to the top of the file, or are we testing importing it as well?

Reply:
All of the pickle tests import in the test itself. I'm not sure if there's a particular reason for that, but I think we can leave it for now. We should change it in all test modules if we do decide to change it though.